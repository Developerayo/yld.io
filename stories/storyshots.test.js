import initStoryshots from '@storybook/addon-storyshots'
import { imageSnapshot } from '@storybook/addon-storyshots-puppeteer'

const { STATIC_BUILD_PATH } = process.env

const beforeScreenshot = async page => {
  return new Promise(resolve =>
    setTimeout(() => {
      resolve()
    }, 600)
  )
}

initStoryshots({
  suite: 'Image storyshots',
  test: imageSnapshot({
    storybookUrl: STATIC_BUILD_PATH
      ? `file://${STATIC_BUILD_PATH}`
      : 'http://localhost:6006',
    beforeScreenshot
  })
})
