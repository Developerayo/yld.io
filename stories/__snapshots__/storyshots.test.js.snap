// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots AnimatedLink AnimatedLink 1`] = `
<a
  className="sc-bdVaJa jDdHMZ"
  href="/undefined"
  onClick={[Function]}
  onMouseEnter={[Function]}
>
  here
</a>
`;

exports[`Storyshots AnimatedLink CardHeader 1`] = `
<header
  className="sc-htpNat mcGtW"
>
  here
</header>
`;

exports[`Storyshots AnimatedLink PosterImage 1`] = `
<div
  className="sc-bwzfXH fmUeyT"
>
  here
</div>
`;

exports[`Storyshots Compact Video Link Dark bg 1`] = `
<section
  className="sc-chPdSV gtkdHC"
>
  <div
    className="sc-ifAKCX lcGsow"
  >
    <div
      className="sc-VigVT gnEyun sc-ifAKCX bTIbjp"
      width={
        Array [
          1,
          1,
          1,
          1,
          0.5,
          0.3333333333333333,
        ]
      }
    >
      <div>
        <div
          className="sc-EHOje cUjftw"
        >
          <div
            className="sc-bZQynM hHdFtZ"
          >
            <a
              className="sc-jzJRlG ioCwbZ"
              color="#fff"
              href="https://www.youtube.com/watch?v=IeuuBKBb4Wg"
              opacity={0.5}
              rel="noopener noreferrer"
              target="_blank"
            >
              <div
                className="sc-cSHVUG kVZMia"
              >
                <div
                  className="sc-EHOje cXsIeh"
                >
                  <img
                    alt="Play button"
                    className="sc-jTzLTM hTdtvM"
                    src="test-file-stub"
                  />
                </div>
                <p
                  className="sc-fjdhpX jbghdt"
                  color="#fff"
                  opacity={0.5}
                >
                  Manipulating the Web Audio API with JSX and Custom Renderers - James Wright
                </p>
              </div>
            </a>
          </div>
        </div>
      </div>
    </div>
  </div>
</section>
`;

exports[`Storyshots Compact Video Link Grey bg 1`] = `
<section
  className="sc-kgoBCf JSEKb"
>
  <div
    className="sc-ifAKCX lcGsow"
  >
    <div
      className="sc-VigVT gnEyun sc-ifAKCX bTIbjp"
      width={
        Array [
          1,
          1,
          1,
          1,
          0.5,
          0.3333333333333333,
        ]
      }
    >
      <div>
        <div
          className="sc-EHOje cUjftw"
        >
          <div
            className="sc-bZQynM hHdFtZ"
          >
            <a
              className="sc-jzJRlG ioCwbZ"
              color="#333333"
              href="https://www.youtube.com/watch?v=IeuuBKBb4Wg"
              opacity={1}
              rel="noopener noreferrer"
              target="_blank"
            >
              <div
                className="sc-cSHVUG kVZMia"
              >
                <div
                  className="sc-EHOje cXsIeh"
                >
                  <img
                    alt="Play button"
                    className="sc-jTzLTM hTdtvM"
                    src="test-file-stub"
                  />
                </div>
                <p
                  className="sc-fjdhpX hTLxIn"
                  color="#333333"
                  opacity={1}
                >
                  Manipulating the Web Audio API with JSX and Custom Renderers - James Wright
                </p>
              </div>
            </a>
          </div>
        </div>
      </div>
    </div>
  </div>
</section>
`;

exports[`Storyshots Compact Video Link Multiple components 1`] = `
<div
  className="sc-ifAKCX lcGsow"
>
  <div
    className="sc-VigVT gnEyun sc-ifAKCX bTIbjp"
    width={
      Array [
        1,
        1,
        1,
        1,
        0.5,
        0.3333333333333333,
      ]
    }
  >
    <div>
      <div
        className="sc-EHOje cUjftw"
      >
        <div
          className="sc-bZQynM hHdFtZ"
        >
          <a
            className="sc-jzJRlG ioCwbZ"
            color="#333333"
            href="https://www.youtube.com/watch?v=IeuuBKBb4Wg"
            opacity={1}
            rel="noopener noreferrer"
            target="_blank"
          >
            <div
              className="sc-cSHVUG kVZMia"
            >
              <div
                className="sc-EHOje cXsIeh"
              >
                <img
                  alt="Play button"
                  className="sc-jTzLTM hTdtvM"
                  src="test-file-stub"
                />
              </div>
              <p
                className="sc-fjdhpX hTLxIn"
                color="#333333"
                opacity={1}
              >
                Manipulating the Web Audio API with JSX and Custom Renderers - James Wright
              </p>
            </div>
          </a>
        </div>
      </div>
    </div>
  </div>
  <div
    className="sc-VigVT gnEyun sc-ifAKCX bTIbjp"
    width={
      Array [
        1,
        1,
        1,
        1,
        0.5,
        0.3333333333333333,
      ]
    }
  >
    <div>
      <div
        className="sc-EHOje cUjftw"
      >
        <div
          className="sc-bZQynM hHdFtZ"
        >
          <a
            className="sc-jzJRlG ioCwbZ"
            color="#333333"
            href="https://www.youtube.com/watch?v=IeuuBKBb4Wg"
            opacity={1}
            rel="noopener noreferrer"
            target="_blank"
          >
            <div
              className="sc-cSHVUG kVZMia"
            >
              <div
                className="sc-EHOje cXsIeh"
              >
                <img
                  alt="Play button"
                  className="sc-jTzLTM hTdtvM"
                  src="test-file-stub"
                />
              </div>
              <p
                className="sc-fjdhpX hTLxIn"
                color="#333333"
                opacity={1}
              >
                Some smaller title
              </p>
            </div>
          </a>
        </div>
      </div>
    </div>
  </div>
  <div
    className="sc-VigVT gnEyun sc-ifAKCX bTIbjp"
    width={
      Array [
        1,
        1,
        1,
        1,
        0.5,
        0.3333333333333333,
      ]
    }
  >
    <div>
      <div
        className="sc-EHOje cUjftw"
      >
        <div
          className="sc-bZQynM hHdFtZ"
        >
          <a
            className="sc-jzJRlG ioCwbZ"
            color="#333333"
            href="https://www.youtube.com/watch?v=IeuuBKBb4Wg"
            opacity={1}
            rel="noopener noreferrer"
            target="_blank"
          >
            <div
              className="sc-cSHVUG kVZMia"
            >
              <div
                className="sc-EHOje cXsIeh"
              >
                <img
                  alt="Play button"
                  className="sc-jTzLTM hTdtvM"
                  src="test-file-stub"
                />
              </div>
              <p
                className="sc-fjdhpX hTLxIn"
                color="#333333"
                opacity={1}
              >
                Some smaller title
              </p>
            </div>
          </a>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`Storyshots Compact Video Link Single component 1`] = `
<div
  className="sc-ifAKCX lcGsow"
>
  <div
    className="sc-VigVT gnEyun sc-ifAKCX bTIbjp"
    width={
      Array [
        1,
        1,
        1,
        1,
        0.5,
        0.3333333333333333,
      ]
    }
  >
    <div>
      <div
        className="sc-EHOje cUjftw"
      >
        <div
          className="sc-bZQynM hHdFtZ"
        >
          <a
            className="sc-jzJRlG ioCwbZ"
            color="#333333"
            href="https://www.youtube.com/watch?v=IeuuBKBb4Wg"
            opacity={1}
            rel="noopener noreferrer"
            target="_blank"
          >
            <div
              className="sc-cSHVUG kVZMia"
            >
              <div
                className="sc-EHOje cXsIeh"
              >
                <img
                  alt="Play button"
                  className="sc-jTzLTM hTdtvM"
                  src="test-file-stub"
                />
              </div>
              <p
                className="sc-fjdhpX hTLxIn"
                color="#333333"
                opacity={1}
              >
                Manipulating the Web Audio API with JSX and Custom Renderers - James Wright
              </p>
            </div>
          </a>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`Storyshots CookieBanner CookieBanner 1`] = `
<div
  className="sc-kGXeez emiswm"
>
  <p
    className="sc-kpOJdX bfqzRn"
  >
    By continuing to use our site you agree to the use of cookies in accordance with our
     
    <a
      className="sc-ckVGcZ kYmQbD"
      href="/cookie-policy"
      onClick={[Function]}
      onMouseEnter={[Function]}
    >
      Cookie
    </a>
    ,
     
    <a
      className="sc-ckVGcZ kYmQbD"
      href="/privacy-policy"
      onClick={[Function]}
      onMouseEnter={[Function]}
    >
      Privacy
    </a>
     and
     
    <a
      className="sc-ckVGcZ kYmQbD"
      href="/data-retention-policy"
      onClick={[Function]}
      onMouseEnter={[Function]}
    >
      Data-retention
    </a>
     
    policies.
  </p>
  <button
    className="sc-dxgOiQ frUlkP"
  >
    I agree
  </button>
</div>
`;

exports[`Storyshots Form Button 1`] = `
<button
  className="sc-kEYyzF idKjMa"
>
  Click Me
</button>
`;

exports[`Storyshots Form Checkbox 1`] = `
<section
  className="sc-iAyFgw dPNDpn"
>
  <input
    className="sc-jKJlTe kdMWuV"
    id="checkbox"
    name="checkbox"
    type="checkbox"
  />
  <label
    htmlFor="checkbox"
  >
    Label
  </label>
</section>
`;

exports[`Storyshots Form Field 1`] = `
<section
  className="sc-kkGfuU ctTVkZ"
>
  <input
    className="sc-eNQAEJ bkUgEZ"
    placeholder="test"
    type="text"
  />
</section>
`;

exports[`Storyshots Form Fieldset 1`] = `
<section
  className="sc-iAyFgw dPNDpn"
>
  <input
    className="sc-eNQAEJ bkUgEZ"
    placeholder="test"
    type="text"
  />
</section>
`;

exports[`Storyshots Form Input 1`] = `
<input
  className="sc-eNQAEJ bkUgEZ"
  placeholder="test"
  type="text"
/>
`;

exports[`Storyshots Form Label 1`] = `
<label
  className="sc-hMqMXs bMyMmr"
>
  Label
</label>
`;

exports[`Storyshots Standalone Video Link Dark bg 1`] = `
<section
  className="sc-chPdSV gtkdHC"
>
  <div
    className="sc-ifAKCX lcGsow"
  >
    <div
      className="sc-VigVT gnEyun sc-ifAKCX bTIbjp"
      width={
        Array [
          1,
          1,
          1,
          1,
          0.5,
          0.3333333333333333,
        ]
      }
    >
      <div
        className="sc-kAzzGY hLHFQQ"
      >
        <div
          className="sc-EHOje cUjftw"
        >
          <div
            className="sc-bZQynM jmIaZk"
          >
            <a
              className="sc-jzJRlG ioCwbZ"
              color="#fff"
              href="https://www.youtube.com/watch?v=IeuuBKBb4Wg"
              opacity={0.5}
              rel="noopener noreferrer"
              target="_blank"
            >
              <div
                className="sc-cSHVUG kVZMia"
              >
                <div
                  className="sc-EHOje cXsIeh"
                >
                  <img
                    alt="Play button"
                    className="sc-jTzLTM hTdtvM"
                    src="test-file-stub"
                  />
                </div>
                <p
                  className="sc-fjdhpX jbghdt"
                  color="#fff"
                  opacity={0.5}
                >
                  Manipulating the Web Audio API with JSX and Custom Renderers - James Wright
                </p>
              </div>
            </a>
          </div>
        </div>
      </div>
    </div>
  </div>
</section>
`;

exports[`Storyshots Standalone Video Link Grey bg 1`] = `
<section
  className="sc-kgoBCf JSEKb"
>
  <div
    className="sc-ifAKCX lcGsow"
  >
    <div
      className="sc-VigVT gnEyun sc-ifAKCX bTIbjp"
      width={
        Array [
          1,
          1,
          1,
          1,
          0.5,
          0.3333333333333333,
        ]
      }
    >
      <div
        className="sc-kAzzGY hLHFQQ"
      >
        <div
          className="sc-EHOje cUjftw"
        >
          <div
            className="sc-bZQynM jmIaZk"
          >
            <a
              className="sc-jzJRlG ioCwbZ"
              color="#333333"
              href="https://www.youtube.com/watch?v=IeuuBKBb4Wg"
              opacity={1}
              rel="noopener noreferrer"
              target="_blank"
            >
              <div
                className="sc-cSHVUG kVZMia"
              >
                <div
                  className="sc-EHOje cXsIeh"
                >
                  <img
                    alt="Play button"
                    className="sc-jTzLTM hTdtvM"
                    src="test-file-stub"
                  />
                </div>
                <p
                  className="sc-fjdhpX hTLxIn"
                  color="#333333"
                  opacity={1}
                >
                  Manipulating the Web Audio API with JSX and Custom Renderers - James Wright
                </p>
              </div>
            </a>
          </div>
        </div>
      </div>
    </div>
  </div>
</section>
`;

exports[`Storyshots Standalone Video Link Multiple components 1`] = `
<div
  className="sc-ifAKCX lcGsow"
>
  <div
    className="sc-VigVT gnEyun sc-ifAKCX bTIbjp"
    width={
      Array [
        1,
        1,
        1,
        1,
        0.5,
        0.3333333333333333,
      ]
    }
  >
    <div
      className="sc-kAzzGY hLHFQQ"
    >
      <div
        className="sc-EHOje cUjftw"
      >
        <div
          className="sc-bZQynM jmIaZk"
        >
          <a
            className="sc-jzJRlG ioCwbZ"
            color="#333333"
            href="https://www.youtube.com/watch?v=IeuuBKBb4Wg"
            opacity={1}
            rel="noopener noreferrer"
            target="_blank"
          >
            <div
              className="sc-cSHVUG kVZMia"
            >
              <div
                className="sc-EHOje cXsIeh"
              >
                <img
                  alt="Play button"
                  className="sc-jTzLTM hTdtvM"
                  src="test-file-stub"
                />
              </div>
              <p
                className="sc-fjdhpX hTLxIn"
                color="#333333"
                opacity={1}
              >
                Manipulating the Web Audio API with JSX and Custom Renderers - James Wright
              </p>
            </div>
          </a>
        </div>
      </div>
    </div>
  </div>
  <div
    className="sc-VigVT gnEyun sc-ifAKCX bTIbjp"
    width={
      Array [
        1,
        1,
        1,
        1,
        0.5,
        0.3333333333333333,
      ]
    }
  >
    <div
      className="sc-kAzzGY hLHFQQ"
    >
      <div
        className="sc-EHOje cUjftw"
      >
        <div
          className="sc-bZQynM jmIaZk"
        >
          <a
            className="sc-jzJRlG ioCwbZ"
            color="#333333"
            href="https://www.youtube.com/watch?v=IeuuBKBb4Wg"
            opacity={1}
            rel="noopener noreferrer"
            target="_blank"
          >
            <div
              className="sc-cSHVUG kVZMia"
            >
              <div
                className="sc-EHOje cXsIeh"
              >
                <img
                  alt="Play button"
                  className="sc-jTzLTM hTdtvM"
                  src="test-file-stub"
                />
              </div>
              <p
                className="sc-fjdhpX hTLxIn"
                color="#333333"
                opacity={1}
              >
                Some smaller title
              </p>
            </div>
          </a>
        </div>
      </div>
    </div>
  </div>
  <div
    className="sc-VigVT gnEyun sc-ifAKCX bTIbjp"
    width={
      Array [
        1,
        1,
        1,
        1,
        0.5,
        0.3333333333333333,
      ]
    }
  >
    <div
      className="sc-kAzzGY hLHFQQ"
    >
      <div
        className="sc-EHOje cUjftw"
      >
        <div
          className="sc-bZQynM jmIaZk"
        >
          <a
            className="sc-jzJRlG ioCwbZ"
            color="#333333"
            href="https://www.youtube.com/watch?v=IeuuBKBb4Wg"
            opacity={1}
            rel="noopener noreferrer"
            target="_blank"
          >
            <div
              className="sc-cSHVUG kVZMia"
            >
              <div
                className="sc-EHOje cXsIeh"
              >
                <img
                  alt="Play button"
                  className="sc-jTzLTM hTdtvM"
                  src="test-file-stub"
                />
              </div>
              <p
                className="sc-fjdhpX hTLxIn"
                color="#333333"
                opacity={1}
              >
                Some smaller title
              </p>
            </div>
          </a>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`Storyshots Standalone Video Link Single component 1`] = `
<div
  className="sc-ifAKCX lcGsow"
>
  <div
    className="sc-VigVT gnEyun sc-ifAKCX bTIbjp"
    width={
      Array [
        1,
        1,
        1,
        1,
        0.5,
        0.3333333333333333,
      ]
    }
  >
    <div
      className="sc-kAzzGY hLHFQQ"
    >
      <div
        className="sc-EHOje cUjftw"
      >
        <div
          className="sc-bZQynM jmIaZk"
        >
          <a
            className="sc-jzJRlG ioCwbZ"
            color="#333333"
            href="https://www.youtube.com/watch?v=IeuuBKBb4Wg"
            opacity={1}
            rel="noopener noreferrer"
            target="_blank"
          >
            <div
              className="sc-cSHVUG kVZMia"
            >
              <div
                className="sc-EHOje cXsIeh"
              >
                <img
                  alt="Play button"
                  className="sc-jTzLTM hTdtvM"
                  src="test-file-stub"
                />
              </div>
              <p
                className="sc-fjdhpX hTLxIn"
                color="#333333"
                opacity={1}
              >
                Manipulating the Web Audio API with JSX and Custom Renderers - James Wright
              </p>
            </div>
          </a>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`Storyshots Typography DisplayTitle 1`] = `
<h2
  className="sc-htoDjs kxmtHS"
>
  Section Title
</h2>
`;

exports[`Storyshots Typography DisplayTitle muted 1`] = `
<h2
  className="sc-htoDjs jfNtmJ"
  muted={true}
>
  Section Title muted
</h2>
`;

exports[`Storyshots Typography DisplayTitle regular 1`] = `
<h2
  className="sc-htoDjs kxmtHS"
>
  Section Title regular
</h2>
`;

exports[`Storyshots Typography DisplayTitle reverse 1`] = `
<h2
  className="sc-htoDjs hraXzN"
>
  Section Title reverse
</h2>
`;

exports[`Storyshots Typography Large Body 1`] = `
<h4
  className="sc-iwsKbI bpHRdi"
>
  Large Body
</h4>
`;

exports[`Storyshots Typography Large Body muted 1`] = `
<h4
  className="sc-iwsKbI eEBOyF"
  muted={true}
>
  Large Body
</h4>
`;

exports[`Storyshots Typography Large Body regular 1`] = `
<h4
  className="sc-iwsKbI bpHRdi"
>
  Large Body
</h4>
`;

exports[`Storyshots Typography Large Body reverse 1`] = `
<h4
  className="sc-iwsKbI kgFENl"
>
  Large Body
</h4>
`;

exports[`Storyshots Typography SectionTitle 1`] = `
<h2
  className="sc-gzVnrw gForec"
>
  Section Title
</h2>
`;

exports[`Storyshots Typography SectionTitle muted 1`] = `
<h2
  className="sc-gzVnrw fhweBi"
  muted={true}
>
  Section Title muted
</h2>
`;

exports[`Storyshots Typography SectionTitle regular 1`] = `
<h2
  className="sc-gzVnrw gForec"
>
  Section Title regular
</h2>
`;

exports[`Storyshots Typography SectionTitle reverse 1`] = `
<h2
  className="sc-gzVnrw kfMiWA"
>
  Section Title reverse
</h2>
`;

exports[`Storyshots Typography Small Title 1 1`] = `
<p
  className="sc-gZMcBi dqQpFk"
>
  Small Title
</p>
`;

exports[`Storyshots Typography Small Title 1 muted 1`] = `
<p
  className="sc-gZMcBi hWsGll"
  muted={true}
>
  Small Title
</p>
`;

exports[`Storyshots Typography Small Title 1 noMargin 1`] = `
<p
  className="sc-gZMcBi dqQpFk"
>
  Small Title
</p>
`;

exports[`Storyshots Typography Small Title 1 reverse 1`] = `
<p
  className="sc-gZMcBi cSVRYM"
>
  Small Title
</p>
`;

exports[`Storyshots Typography Small Title 2 1`] = `
<p
  className="sc-gqjmRU ewbdOb"
>
  Small Title
</p>
`;

exports[`Storyshots Typography Small Title 2 noBottom 1`] = `
<p
  className="sc-gqjmRU ewbdOb"
>
  Small Title
</p>
`;

exports[`Storyshots Typography Small Title 2 noMargin 1`] = `
<p
  className="sc-gqjmRU ewbdOb"
>
  Small Title
</p>
`;

exports[`Storyshots Typography Small Title 2 noTop 1`] = `
<p
  className="sc-gqjmRU ewbdOb"
>
  Small Title
</p>
`;

exports[`Storyshots Typography Small Title 2 noWrap 1`] = `
<p
  className="sc-gqjmRU ewbdOb"
>
  Small Title
</p>
`;

exports[`Storyshots Typography Small Title 2 reverse 1`] = `
<p
  className="sc-gqjmRU dgTeFt"
>
  Small Title
</p>
`;
